ui/component/or-attribute-card/package.json

```diff
@@ -23,6 +23,8 @@
     "@openremote/or-attribute-picker": "workspace:*",
     "@openremote/or-chart": "workspace:*",
     "@openremote/or-mwc-components": "workspace:*",
+    "chart.js": "^3.6.0",
+    "chartjs-adapter-moment": "^1.0.0",
```

Comment:
Why is chart.js added here?
Is this component still using chart.js and not echarts?
If so, when will it be migrated?